models/sessions/scratch/bigquery/snowplow_web_sessions_this_run.sql

```diff
@@ -233,6 +214,7 @@ select
     -- engagement fields
     c.page_views,
     c.engaged_time_in_s,
+    case when c.page_views >= 2 and c.engaged_time_in_s >= 10 then true else false end engaged,
```

Comment:
Probably worth making this still dependant on heartbeat length, and I think this was an `or`. There's maybe a case to make this a macro so the user can change it but I think we can do that at some later date if someone requests it. 

For some reason the single word name for a Boolean column doesn't sit right with me, but not sure if that's a standard ga4 name so keep it is it is
```suggestion
    case when c.page_views >= 2 or c.engaged_time_in_s/{{ var('snowplow__heartbeat', 5) }} >= 2 then true else false end as is_engaged_session,
```

Could you include the check for any conversion events as well? You should be able to just use the line for the __all_volume column in https://github.com/snowplow/dbt-snowplow-web/pull/194/files , just make sure to only include it if the conversion events is actually set and still works with it isn't.